ddsc/core/remotestore.py

```diff
@@ -318,6 +318,16 @@ def set_hash(self, file_hash, hash_alg):
         self.file_hash = file_hash
         self.hash_alg = hash_alg
 
+    @staticmethod
+    def get_upload_from_json(json_data):
+        if 'current_version' in json_data:
```

Comment:
Style question - why 

```
if good_format1:
  handle_1
else
  if good_format2:
    handle_2
  else:
    fail
```

instead of

```
if good_format1:
  handle_1
elif good_format2:
  handle_2
else:
  fail
```